From cd9fc30d92076562214ec4c319440d4375d1d88a Mon Sep 17 00:00:00 2001
From: Yu-hsin Wang <yuhsingw@google.com>
Date: Mon, 10 Jan 2022 16:36:15 +0800
Subject: [PATCH 110/757] dev-arm: Add missing sim_objects of VExpressFastmodel

Change-Id: Ic6a9e5f1381c6c6412faa6d19f1448ca0e08b1e7
Reviewed-on: https://gem5-review.googlesource.com/c/public/gem5/+/55304
Reviewed-by: Gabe Black <gabe.black@gmail.com>
Maintainer: Gabe Black <gabe.black@gmail.com>
Reviewed-by: Andreas Sandberg <andreas.sandberg@arm.com>
Maintainer: Andreas Sandberg <andreas.sandberg@arm.com>
Tested-by: kokoro <noreply+kokoro@google.com>
---
 src/dev/arm/SConscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/dev/arm/SConscript b/src/dev/arm/SConscript
index e74b190..e59dc94 100644
--- a/src/dev/arm/SConscript
+++ b/src/dev/arm/SConscript
@@ -64,7 +64,7 @@ SimObject('NoMali.py', sim_objects=['NoMaliGpu', 'CustomNoMaliGpu'],
     enums=['NoMaliGpuType'], tags='arm isa')
 SimObject('VirtIOMMIO.py', sim_objects=['MmioVirtIO'], tags='arm isa')
 if env['USE_ARM_FASTMODEL']:
-    SimObject('VExpressFastmodel.py', tags='arm isa')
+    SimObject('VExpressFastmodel.py', sim_objects=[], tags='arm isa')
 
 Source('a9scu.cc', tags='arm isa')
 Source('amba_device.cc', tags='arm isa')
-- 
1.8.3.1

